addons/mail/static/src/core/common/chat_bubble.xml

```diff
@@ -28,7 +28,7 @@
                 <t t-call="mail.message_preview_prefix">
                     <t t-set="message" t-value="thread?.newestPersistentNotEmptyOfAllMessage"/>
                 </t>
-                <t t-esc="thread?.newestPersistentNotEmptyOfAllMessage.inlineBody"/>
+                <t t-esc="previewContent"/>
```

Comment:
What was the use of `previewContent` before this PR if it wasn't used here?

Reply:
Name's poor, but it should only affect the message body, whether false or inline body if there is last message.
There was some confusion with showing author prefix, but this should have been managed with `mail.message_preview_prefix`.

I made the changes just now